library/HTMLPurifier/AttrDef/HTML/LinkRel.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+/*
+ * Keywords that are body-ok affect whether link elements are allowed in the body.
+ * The body-ok keywords are dns-prefetch, modulepreload, pingback, preconnect, prefetch, preload, prerender,
+ * and stylesheet.
+ *
+ * https://html.spec.whatwg.org/multipage/links.html#body-ok
+ *
+ * @note We cannot use Enum because multiple values are allowed.
+ */
+class HTMLPurifier_AttrDef_HTML_LinkRel extends HTMLPurifier_AttrDef_HTML5_ARel
```

Comment:
This class definitely should be in `HTMLPurifier_AttrDef_HTML5` namespace - `<link>` elements in the `<body>` are invalid in HTML4 and XHTML specs.